### PR TITLE
ktx: Verify the existence of config file before sourcing

### LINKED ISSUE #2: Error when selecting non-existing config
Right now you can use any string with ktx. It should, at the very least, let you know that that file does not exist.

Example:

     salazar:~ cha$ ktx 
    dev    prod   test
     salazar:~ cha$ eval $(ktx hello-world)
    hello-world salazar:~ cha$ 

Expected result

     salazar:cluster-config cha$ eval $(ktx hello-world)
    File does not exist! Not doing anything.

### PR DESCRIPTION
**Pull request by tylerauerbeck**
Check for the existence of the config file before exporting the KUBECONFIG environment variable

Signed-off-by: Tyler Auerbeck <auerbecktj@gmail.com>

Fixes #2

### CODE REVIEW

File: ktx [hunk 1]
```diff
@@ -15,7 +15,10 @@
 # limitations under the License.
 
 main() {
-    # TODO: ensure the file actually exists.
-    echo "export KUBECONFIG=\"${HOME}/.kube/${1}\""
+    if [ -e ${HOME}/.kube/${1} ];then
+        echo "export KUBECONFIG=\"${HOME}/.kube/${1}\""
+    else
+        echo "echo \"The following file does not exist: ${HOME}/.kube/${1}. Exiting...\""
```
Comment (chuckha) [inline ktx]: Let's add a comment to the top of the function explaining why we have to echo "echo". This could be confusing in future.

File: ktx [hunk 2]
```diff
@@ -15,7 +15,10 @@
 # limitations under the License.
 
 main() {
-    # TODO: ensure the file actually exists.
-    echo "export KUBECONFIG=\"${HOME}/.kube/${1}\""
+    if [ -e ${HOME}/.kube/${1} ];then
```
Comment (chuckha) [inline ktx]: I think these variables should be in quotes

**Comment by tylerauerbeck** [comment]
@chuckha That's a good point, some of this may be confusing after awhile. Added the comment you mentioned and made sure to quote the variables inside of the if statement.

**Comment by chuckha** [comment]
Thank you!

**Pull request merged**